packages/dashboards/src/lib/configurator/services/converters/timeseries/timeseries-metadata-converter.service.spec.ts

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable */
```

Comment:
it may be good to add the rule that you're disabling here if possible

```suggestion
/* eslint-disable @typescript-eslint/no-unused-vars */
```